### ISSUE TITLE
Avoid bower install warnings

### ISSUE THREAD

**Issue by SirRagnar** [terebentina/sco.js#24]
With the [1.0.2](https://github.com/terebentina/sco.js/releases/tag/1.0.2) version there is some warnings when the package is installed through Bower:

```shell
$ bower install sco.js --save
bower sco.js#*              not-cached git://github.com/terebentina/sco.js.git#*
bower sco.js#*                 resolve git://github.com/terebentina/sco.js.git#*
bower sco.js#*                download https://github.com/terebentina/sco.js/archive/1.0.2.tar.gz
bower sco.js#*                 extract archive.tar.gz
bower sco.js#*            invalid-meta sco.js is missing "main" entry in bower.json
bower sco.js#*            invalid-meta sco.js is missing "ignore" entry in bower.json
bower sco.js#*                resolved git://github.com/terebentina/sco.js.git#1.0.2
bower sco.js#~1.0.2            install sco.js#1.0.2

sco.js#1.0.2 public\lib\sco.js
```

Would be great if you could add a ``bower.json`` with the `ignore` statement. Something like this:

```json
...
  "ignore": [
    "**/.*",
    "tests"
  ],
...
```

**Comment by bmrainc** [comment]
is the problem solved?

**Comment by SirRagnar** [comment]
No, it isn't. I could push a PR if @terebentina wants.

**Comment by terebentina** [comment]
Yes, please do. I am happy for any PR or maintainers. I haven't worked with jquery in a good while and I don't really have time to maintain sco.js atm

**Comment by SirRagnar** [comment]
The version property in bower.json should be upgraded to match next sco version.

**Comment by terebentina** [comment]
Ok, I've had time to look at this issue. Bower's future is very unclear now (only 1 maintainer with no funding who works on bower in his spare time) but this isn't really related. Just a warning that you may not want to use bower, if possible.

Anyway, the [bower spec for main](https://github.com/bower/spec/blob/master/json.md#main) says I can use at most 1 js file in main and at most 1 css file. This means I should build all of sco.js modules into a single build file and use that. It's not really what I wanted for it as I doubt you're going to need all of its modules at once in the same project. I feel it's better to only include the module you need.
Anyway, I cannot accept the PR which lists just sco.message.js in main.
I could add the ignore bit, no problem, but could you let me know why you need any of these options? Is your building script failing or is there any other problem other than the 2 warnings?

**Comment by SirRagnar** [comment]
You are right about bower spec for main, sorry for not being aware before push the PR.

I use [main-bower-files](https://www.npmjs.com/package/main-bower-files) node module in my build automated tasks and they inject bower dependences into my html files. I use it too in my [Karma](http://karma-runner.github.io/) conf for automated test. If there is no main attribute in the `bower.json` dependece, _main-bower-files_ can't figure out what's to inject. In libraries where there are no main property or has many js/css files in it, you need to include an _"override"_  property in your own `bower.json` file. I have something like this in my `bower.json`:
```json
  "overrides": {
    "sco.js": {
      "main": [
        "js/sco.message.js",
        "css/scojs.css"
      ]
    }
 }
```

Anyway,an _"ignore"_ property in `bower.json` will do SCO dependence lighter.
